page/profile/index.html

```diff
@@ -0,0 +1,19 @@
+<Page title="Profile">
+    <Layout page='profile'>
+        <div class="profile-page">
+            <div class="container page">
+                <div class="row">
+                    <div class="col-md-6 offset-md-3 col-xs-12">
```

Comment:
Can you update this to same structure as in sign-up - i.e. remote 3 of the divs and change the remaining one ro class="grid-row"